### ISSUE TITLE
Result is 4x4 "big pixel" image

### ISSUE THREAD

**Issue by OndrejTexler** [grey-eye/talking-heads#35]
Hello! 

I use the latest code without any changes and I am not able to get any decent results. On subset of 3000 .vid files after 7 epochs all results look like this one:
![20190906_021456209527_x_hat](https://user-images.githubusercontent.com/22987428/64393201-2d9a1f80-d005-11e9-81aa-93dde79b4d3d.png)
The losses are following:
`Epoch 8: [37/1000] | Time: 0:00:00.557204 | Loss_E_G = 0.38158226 Loss_D = 1.00000143`

Do you have any idea why I am getting those completely wrong results? ... Thank you!

**Comment by OWLonoak** [comment]
I meet the same issues.Every time I set FEEDFOWARD to False，the results will be several big pixels.


------------------ Original ------------------

**Comment by OWLonoak** [comment]
Is that cause by learning rate setting too low？

**Comment by OndrejTexler** [comment]
Great, thank you for a good tip! Setting FEED_FORWARD to False disables the match loss in the `forward` method of `LossEG`. According to the original paper, match loss should be used during the meta training stage; although, I would assume that not using match loss would not cause results to be that bad.

**Comment by castelo-software** [comment]
This is a known issue. I'm not working to solve it at the moment.
You can check the corresponding thread: https://github.com/grey-eye/talking-heads/issues/26
